<?xml version="1.0"?>
<launch>
  <group ns="guzheng">
    <include file="$(find music_perception)/launch/guzheng_onsets.launch" />
    <node name="onset_to_marker" pkg="music_perception" type="onset_to_marker.py" />
  </group>
</launch>
